language/e2e-testsuite/src/tests/verify_txn.rs

```diff
@@ -479,6 +479,23 @@ fn verify_expiration_time() {
         executor.execute_transaction(txn).status(),
         StatusCode::TRANSACTION_EXPIRED
     );
+
+    let txn = transaction_test_helpers::get_test_signed_transaction(
```

Comment:
Could you add a comment explaining that the 10 is deliberately chosen to try to trigger `SEQUENCE_NUMBER_TOO_NEW`?